airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/2af123bf-0aaf-4e0d-9784-cb497f23741a.json

```diff
@@ -2,7 +2,7 @@
   "sourceDefinitionId": "2af123bf-0aaf-4e0d-9784-cb497f23741a",
   "name": "Appstore",
   "dockerRepository": "airbyte/source-appstore-singer",
-  "dockerImageTag": "0.2.4",
+  "dockerImageTag": "0.2.5",
```

Comment:
According to the latest changes, we no longer need to change the versions in the `json` files, it will be enough to change the versions only in the `airbyte-config/init/src/main/resources/seed/source_definitions.yaml` file.
Then it will also be necessary to run a command that will update the versions in the `airbyte-config/init/src/main/resources/seed/source_spec.yaml` file, but this is a little later, when the PR receives the approvals and we publish all the connectors.